app/services/data_migrations/remove_one_login_pre_release_banners_feature_flag.rb

```diff
@@ -0,0 +1,10 @@
+module DataMigrations
+  class RemoveOneLoginPreReleaseBannersFeatureFlag
+    TIMESTAMP = 20250120150435
+    MANUAL_RUN = false
+
+    def change
+      Feature.where(name: :one_login_candidate_sign_in).delete_all
```

Comment:
```suggestion
      Feature.where(name: :one_login_pre_release_banners).delete_all
```

Should we be removing the banners feature rather than the sign in feature?

Reply:
omg, I'm losing my mind.